content/post/career-ladders-part-3.md

```diff
@@ -0,0 +1,57 @@
+---
+author: Caitlin Moorman
+title: "How to Use Your Career Ladder"
+cover: ""
+draft: true
+publishDate: "2019-01-20"
+tags:
+ - analytics
+ - org
+ - teams
+ - management
+---
+
+Providing [clarity and structure](https://www.locallyoptimistic.com/post/career-ladders-part-2/) around expectations empowers your team members to decide what skills and competencies to focus on improving. However, writing the career ladder is only the beginning of effectively using it - to get the most value, you have to communicate it to your team about, and use it as a jumping off point for specific, growth-oriented development conversations.
```

Comment:
May want to link to the first post as well in case someone lands on this post directly.